polyfill/lib/ecmascript.mjs

```diff
@@ -982,13 +982,13 @@ export const ES = ObjectAssign({}, ES2020, {
       } else {
         any = true;
       }
-      if (BUILTIN_FIELDS.has(property)) {
+      if (BUILTIN_FIELDS.has(property) && !CALENDAR_FIELDS.has(property)) {
```

Comment:
Nothing against your code in particular, but this line makes me think that too much has been bolted on to `ToRecord` already, making it too confusing, and I'd like to refactor it. However since this part is already different than the spec text, that can be done later.

Reply:
agreed